include/game.h

```diff
@@ -55,6 +56,8 @@ typedef struct GameState {
     /* 0x9C */ UNK_TYPE1 unk_9C[0x3];
     /* 0x9F */ u8 running;
     /* 0xA0 */ s32 unk_A0;
+    /* 0xA4 */ UNK_TYPE1 unk_A4;
+    /* 0xA8 */ Controller controller;
 } GameState; // size >= 0xA4, size <= 0xE0
```

Comment:
Can you update the size comment now as well. Looks like we now know it is exactly `0xE0`